src/crate/client/sqlalchemy/tests/create_table_test.py

```diff
@@ -232,3 +232,55 @@ class DummyTable(self.Base):
             a = sa.Column(Geopoint, crate_index=False)
         with self.assertRaises(sa.exc.CompileError):
             self.Base.metadata.create_all(bind=self.engine)
+
+    def test_column_server_default(self):
```

Comment:
Just a small suggestion on matters of "naming things" .
```suggestion
    def test_column_server_default_text_func(self):
```